### DISCUSSION TITLE
Include bluetooth audio-in. Parallel in the mix.

### DISCUSSION THREAD

**Discussion by Retrofreak** [rern/rAudio#1480] [Feature Requests]
Hi, is it possible to receive an audio signal from an external source via Bluetooth. Include it in e.g. alsamixer on the console. (I would like to listen to my Echo Dot 5 at the same time via the hi-fi system). It would be great if that were possible ;-)

**Comment by kan-ibal** [comment]
Bluetooth works in "Audio - Receiver / Sender" mode.
